### ISSUE TITLE
New hired-ness setting not getting propagated? 

### ISSUE THREAD

**Issue by ryan-cranfill** [ideo/Meetbot#5]
Hey @jzanzig @lisamnash was poking through your code to see how it works and noticed this line: https://github.com/ideo/Meetbot/blob/master/triad_optimization.py#L111

Looks like you have a setting for how long a person is considered a new hire, but the threshold in calculating it is hard coded. apologies if i'm wrong/misinterpreting and adding noise!

**Comment by lisamnash** [comment]
Yeah, it's hard-coded at 180 days. We could change it. Probably lots of stuff that could be changed here...!
